main.py

```diff
@@ -300,17 +310,40 @@ def search_key():
                             if not haswon:
                                 #'you are out of turns!'
                                 done=True
-                                text=basicFont.render('GAME OVER', True, BLACK)
+                                textSurf9, textRect9 = text_objects("GAME OVER", numTurnsLeft)

```

Comment:
This text doesn't fit the box

Reply:
![error](https://user-images.githubusercontent.com/40236421/50243418-2f7a6f00-03f3-11e9-957d-15866d0853c4.png)

It fits in mine but I'll change the font size to 20. Would that be fine?

Reply:
LGTM. I still have my doubts on whether this is the best way to do it, but I'm merging this PR. Good work.